### ISSUE TITLE
bug: `cap run android` blank screen, “Cannot read property ‘triggerEvent’ of undefined”

### ISSUE THREAD

**Issue by emmernme** [ionic-team/capacitor#7048]
### Capacitor Version
```
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 5.5.1
  @capacitor/core: 5.5.1
  @capacitor/android: 5.5.1
  @capacitor/ios: 5.5.1

Installed Dependencies:

  @capacitor/cli: 5.2.2
  @capacitor/core: 5.5.1
  @capacitor/android: 5.5.1
  @capacitor/ios: 5.5.1

[success] iOS looking great! 👌
[error] Missing <manifest package=""> attribute in app/src/main
```

### Platform(s)
Android - simulator and real device

### Current Behavior
When running `ionic cap run android`, upon the device installing and opening the app, I am presented with a blank screen. The Chrome Inspector logs the following error twice:

```Cannot read property 'triggerEvent' of undefined.```

This issue has persisted across multiple versions of Ionic, now in 5.5.1 with native-run 2.0.0. Running macOS 13.0-14.0

Adding `android:usesCleartextTraffic="true"` to AndroidManifest.xml does not affect this.

### Expected Behavior
App should load as normal, as it does on iOS with no issues. Running the app manually through Android Studio works as expected.

### Code Reproduction

Please see https://github.com/emmernme/capacitor-bug-7048.

### Other Technical Details

`npm --version` output: 8.19.4
`node --version` output: v16.20.2

### Additional Context

Referencing https://forum.ionicframework.com/t/cap-run-android-blank-screen-cannot-read-property-triggerevent-of-undefined/233496

**Comment by jcesarmobile** [comment]
I can't reproduce on the provided app, what I get is `Uncaught (in promise) String resource ID #0x0` and the app hangs on the splash screen, that's because `cordova-plugin-ionic` is not configured.
If I manually hide the splash screen then the app works.
Can you provide another sample app where `cordova-plugin-ionic` is not installed or is configured correctly?

**Comment by emmernme** [comment]
Thank you for responding!
Thanks to your response I was able to remove `cordova-plugin-ionic`, which was left behind when migrating from Cordova to Capacitor. While testing after this change, stumbling upon the `ERR_CLEARTEXT_NOT_PERMITTED`-error in the simulator (an improvement from a blank screen, at least), changing a few things, I managed to solve the issue without fully identifying the cause. I made the following changes:

1. Updating my local global Ionic CLI version from 6.16.3 to 7.1.1.
2. Removed `android:networkSecurityConfig="@xml/network_security_config"` which was unfortunately not copied over to the reproduction repo in the first place - I think this might have been the culprit.

The `network_security_config`-entry was there for unknown reasons, and obviously unnecessary (and also an undefined variable).

I have also pushed a version of the reproduction repo without `cordova-plugin-ionic` to confirm this is indeed now solved.
